node-src/tasks/verify.ts

```diff
@@ -57,7 +57,7 @@ export const publishBuild = async (ctx: Context) => {
       id,
       input: {
         ...(onlyStoryFiles && { onlyStoryFiles }),
-        ...(onlyStoryNames && { onlyStoryNames: [].concat(onlyStoryNames) }),
+        ...(onlyStoryNames && { onlyStoryNames: [onlyStoryNames] }),
```

Comment:
@ethriel3695 This is a functional change. Why was this changed, and why is it part of this PR?

Reply:
@ghengeveld I changed it because TypeScript was yelling at me about it.
I'll revert.